nestkernel/nest_names.cpp

```diff
@@ -484,6 +484,7 @@ const Name U_std( "U_std" );
 const Name U_upper( "U_upper" );
 const Name update( "update" );
 const Name update_node( "update_node" );
+const Name use_gid( "use_gid" );
```

Comment:
This name should refer to the fact that the gid is used in the filename. As it is now, this does not become clear from the variable name